### PR TITLE
Tracking queries.

### PR DESCRIPTION
**Pull request by akash1810**

### CODE REVIEW

**Comment by theefer** [comment]
:+1:

File: kahuna/public/js/search/query.js [hunk 1]
```diff
@@ -49,6 +52,11 @@ query.controller('SearchQueryCtrl',
             // FIXME: broken for 'your uploads'
             // FIXME: + they triggers filter $watch and $state.go (breaks history)
             ctrl.filter[key] = valOrUndefined(newVal);
+
+            // don't track changes to `query` as it would trigger on every keypress
+            if (key !== 'query') {
+                track.success('Query Change', { field: key, value: newVal });
```
Comment (jamesgorrie) [inline kahuna/public/js/search/query.js]: `Query change` for consistency

**Comment by jamesgorrie** [comment]
This is going to hammer our Mixpanel quota no?

File: kahuna/public/js/search/query.js (hunk 1 above)
Comment (akash1810) [inline kahuna/public/js/search/query.js]: https://github.com/guardian/grid/pull/967

**Comment by akash1810** [comment]
> This is going to hammer our Mixpanel quota no?

Depends on how much the filters are used (we're not tracking the free text search).

**Pull request merged**